bin/copy_ci_build_to_folder.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
```

Comment:
same for this script - do we still need this?

Reply:
Yep, we still need this one - this particular script renames the ZIP artifact created on CI to `confluentinc-kafka-connect-s3-UdxStreamPartitioner.zip` which means we can reference a constant filename in [the terraform config](https://github.com/Urban-Data-Collective/udx-infra/blob/UDX-1216-create-custom-partitioner-for-msk-connect-connector/terraform/modules/data-lake/variables.tf#L31)